changelogs/change_log_Sept2021.md

```diff
@@ -0,0 +1,133 @@
+# EMAP Release
```

Comment:
might be worth using YYYY-MM in filename so files sort naturally by release date